app/src/main/java/com/blogspot/e_kanivets/moneytracker/activity/record/MainActivity.kt

```diff
@@ -0,0 +1,188 @@
+package com.blogspot.e_kanivets.moneytracker.activity.record
+
+import android.content.Intent
+import android.support.v7.app.AppCompatActivity
+import android.widget.TextView
+import com.blogspot.e_kanivets.moneytracker.R
+import com.blogspot.e_kanivets.moneytracker.activity.ReportActivity
+import com.blogspot.e_kanivets.moneytracker.activity.base.BaseDrawerActivity
+import com.blogspot.e_kanivets.moneytracker.adapter.RecordAdapter
+import com.blogspot.e_kanivets.moneytracker.controller.CurrencyController
+import com.blogspot.e_kanivets.moneytracker.controller.FormatController
+import com.blogspot.e_kanivets.moneytracker.controller.PeriodController
+import com.blogspot.e_kanivets.moneytracker.controller.PreferenceController
+import com.blogspot.e_kanivets.moneytracker.controller.data.AccountController
+import com.blogspot.e_kanivets.moneytracker.controller.data.ExchangeRateController
+import com.blogspot.e_kanivets.moneytracker.controller.data.RecordController
+import com.blogspot.e_kanivets.moneytracker.entity.Period
+import com.blogspot.e_kanivets.moneytracker.entity.data.Record
+import com.blogspot.e_kanivets.moneytracker.report.ReportMaker
+import com.blogspot.e_kanivets.moneytracker.ui.AppRateDialog
+import com.blogspot.e_kanivets.moneytracker.ui.presenter.ShortSummaryPresenter
+import com.blogspot.e_kanivets.moneytracker.util.AnswersProxy
+import kotlinx.android.synthetic.main.activity_main.*
+import kotlinx.android.synthetic.main.content_main.*
+import javax.inject.Inject
+
+
+class MainActivity : BaseDrawerActivity() {
+
+    private lateinit var recordList: List<Record>
+    private lateinit var period: Period
+
+    @Inject
+    lateinit var recordController: RecordController
+    @Inject
+    lateinit var rateController: ExchangeRateController
+    @Inject
+    lateinit var accountController: AccountController
+    @Inject
+    lateinit var currencyController: CurrencyController
+    @Inject
+    lateinit var preferenceController: PreferenceController
+    @Inject
+    lateinit var periodController: PeriodController
+    @Inject
+    lateinit var formatController: FormatController
+
+    private lateinit var summaryPresenter: ShortSummaryPresenter
+
+    private lateinit var tvDefaultAccountTitle: TextView
+    private lateinit var tvDefaultAccountSum: TextView
+    private lateinit var tvCurrency: TextView
+
+    override fun getContentViewId(): Int {
```

Comment:
Kotlin way of doing it would be `override fun getContentViewId() = R.layout.activity_main`.